cmd/getcreds/main.go

```diff
@@ -48,46 +50,33 @@ func getUserHomeDir(usr *user.User) (string, error) {
 	return usr.HomeDir, nil
 }
 
-// generateKeyPair uses internal golan functions to be portable
+// generateKeyPair uses internal golang functions to be portable
 // mostly comes from: http://stackoverflow.com/questions/21151714/go-generate-an-ssh-public-key
-func genKeyPair(privateKeyPath string) (string, error) {
+func genKeyPair(privateKeyPath string) (crypto.Signer, string, error) {
 	privateKey, err := rsa.GenerateKey(rand.Reader, RSA_KEY_SIZE)
 	if err != nil {
-		return "", err
+		return nil, "", err
 	}
 
 	//privateKeyPath := BasePath + prefix
```

Comment:
Can you please make comments consistent: have a space between the "//" and the first character.